runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterView.tsx

```diff
@@ -84,9 +86,9 @@ const UsersRegisterView: FC = () => {
                 enqueueSnackbar(translate('Users.Register.View.Events.Error.AcceptFailed'), { variant: 'error' });
             });
 
-    const handleDelete = () => {
-        // to be implemented soon
-    };
+    const handleDelete = () => setShowDeleteDialog(true);
+
+    const sendSelectedUsersToDeleteModal = (): IUser[] => notActivated.allByPage.content.filter((user) => selections.includes(user.id));
```

Comment:
you do use it to pass user's to delete modal but there's nothing about the modal in function body.

rename to sth like getSelectedUsers